frontend/src/scenes/web-analytics/WebDashboard.tsx

```diff
@@ -249,6 +275,57 @@ export const WebTabs = ({
     )
 }
 
+export interface LearnMorePopoverProps {
+    docsURL: PostHogComDocsURL
+    title: string
+    description: string
+}
+
+export const LearnMorePopover = ({ docsURL, title, description }: LearnMorePopoverProps): JSX.Element => {
+    const [isOpen, setIsOpen] = useState(false)
+
+    return (
+        <Popover
+            visible={isOpen}
+            onClickOutside={() => setIsOpen(false)}
+            overlay={
+                <div className="p-4">
+                    <div className="flex flex-row w-full">
+                        <h2 className="flex-1 text-lg font-bold">{title}</h2>
+                        <LemonButton
+                            targetBlank={true}
+                            type="tertiary"
+                            onClick={() => setIsOpen(false)}
+                            size="xsmall"
+                            icon={<IconX />}
+                        />
+                    </div>
+                    <p className="text-sm text-gray-700">{description}</p>
+                    <div className="flex justify-end mt-4">
+                        <LemonButton
+                            to={docsURL}
+                            onClick={() => setIsOpen(false)}
+                            targetBlank={true}
+                            sideIcon={<IconOpenSidebar className="w-4 h-4" />}
```

Comment:
Out of curiosity did you intentionally add the size classes to this icon? Suggests that 1) icon sizes are incorrect on this size button or 2) this icon is the wrong default size

Reply:
This was copied directly from ProductIntroduction.tsx but have removed it